src/components/sky-nav/sky-nav.scss

```diff
@@ -208,11 +208,15 @@ $_max-width: #{unit.strip(layout.$max-width-spread) +
 /**
  * The element that toggles the menu visibility at narrow widths.
  *
- * 1. This prevents us having to set and then override padding on the masthead
+ * 1. Vertical positioning is handled by JS because it's easier to grab the
+ *    height of the header bar from there.
```

Comment:
I'm gonna poke at this a bit, because there's something about overtly setting a specific property via JS that feels like it's overstepping the separation of concerns between CSS and JavaScript a tad?

Some alternative ideas to consider:

1. Could we save `header.clientHeight` to a custom property, and use that and the existing `$control-height` token in a `calc` function to position the element?
2. Could we set the nav bar height to a specific value in CSS so that we know exactly how to position the button? This would likely mean some new restrictions in terms of nav item wrapping but I think it's fair to say "this is optimized for X number of rows maximum" if we need to.

Reply:
Now that I'm looking at this again, I don't see why we can't set a fixed height on the navbar for small screens, which would allow calculating the offset using CSS. Not sure why I didn't see that to begin with, but I'll give that a shot now.

Reply:
Updated!